src/components/task-plan/builder.cjsx

```diff
@@ -130,10 +130,10 @@ module.exports = React.createClass
     {id} = @props
     TaskPlanActions.updateDescription(id, desc)
 
-
+  # can't figure out how to remove this
```

Comment:
Would removing `renderFeedbackNote: ->` and then `if plan.type is 'homework'` and then `{feedbackNote}` work?